docs/.vale.ini

```diff
@@ -14,4 +14,4 @@ BasedOnStyles = expo-docs
 BlockIgnores = (?s) <Terminal(\s|\n)[^/>]*?property
 IgnoredClasses= whitesapce-pre, prose-code
 SkippedScopes = script, style, pre, figure, code, tt, tr, td, span
-TokenIgnores = (\x60[^\n\x60]+\x60), ([^\n]+=[^\n]*), (\+[^\n]+\+), (http[^\n]+\[), \$ yarn, \$ eas build, \$ eas update
\ No newline at end of file
+TokenIgnores = (\x60[^\n\x60]+\x60), ([^\n]+=[^\n]*), (\+[^\n]+\+), (http[^\n]+\[), .github, .gitlab, vscode-, Signing & Capabilities, eas.json, eas-json, .yarn+, yarn: "# yarn", eas+, eas-cli, npx+, Build & deploy, OAuth & Permissions, Languages & Frameworks, typescript, Privacy & Security, Certificates, IDs & Profiles, application/javascript, Settings..., killing..., Cleaning up..., creating..., Downloading..., files..., Motion & Orientation Access,
```

Comment:
We should use Ellipsis character (`…`) instead of three dots in sequence, if such rule is possible:
* https://en.wikipedia.org/wiki/Ellipsis

Reply:
We do have an Ellipses rule (https://github.com/expo/expo/blob/main/docs/.vale/writing-styles/expo-docs/Ellipses.yml) but in cases where it is defined as a code block comment or is file/setting name (for example, **abc...**) the rule is overridden and not looked by Value. Maybe it is because of the regex pattern that I am using.

Adding `...` to TokeIgnores complete ignores it. So great suggestion!

Reply:
We still should replace instances of three dots `...` used in the docs content with the ellipsis character `…`, unless it's written on purpose that way.

Reply:
Makes sense. Apologies, I misunderstood your first comment. I think the purpose of three dots was to denote Ellipses. Let me fix that.

Reply:
@Simek, this now fixed: https://github.com/expo/expo/pull/24593/commits/c3f0916e9c448ae6786f5d40947ec533a802a525. Also, replaced the vale rule with to detect ellipses. In the code examples where we currently use it, it requires to be used so I am going to leave them as it is.